couchdb/values.yaml

```diff
@@ -163,3 +163,21 @@ couchdbConfig:
 # This is used to generate FQDNs for peers when joining the CouchDB cluster.
 dns:
   clusterDomainSuffix: cluster.local
+
+# Adds a sidecar to expose CouchDB metrics as a prometheus endpoint
+# see https://github.com/gesellix/couchdb-prometheus-exporter/blob/master/README.md
+telemetryImage:
+  repository: gesellix/couchdb-prometheus-exporter
+  tag: latest
+  pullPolicy: Always
```

Comment:
I was worried about the impact on deploy times by defaulting to `Always` here, but it's fortunately a tiny image.